invoicing_app/management/commands/update_incomplete_tax_receipts.py

```diff
@@ -0,0 +1,105 @@
+                    reporting_country_code__in=['AR', 'BR']).iterator()
+
+    def update_tax_receipts_that_met_requirements(self):
+        for tax_receipt in self.tax_receipts:
+            po = PaymentOptions.objects.using('default').get(event=tax_receipt.event_id)
+            if tax_receipt.reporting_country_code == 'AR':
+                self.__check_ARG_requirements(tax_receipt, po)
+            else:
+                self.__check_BR_requirements(tax_receipt, po)
+
+    def __check_ARG_requirements(self, tax_receipt, payment_option):
+        # IF ONE FIELD FAILS CHECK REQUIREMENTS, WE CANT CHANGE TO 'PENDING' STATUS
+        # SO ITS POINTLESS TO KEEP CHECKING REST OF FIELDS.
+        for requirement in self.arg_requirements:
+            po_attribute = getattr(payment_option, str(self.tax_to_po_requirement_dict.get(requirement)), '')
+            if self.__check_single_requirement(po_attribute):
+                setattr(tax_receipt, requirement, po_attribute)
+            else:
+                return
+        self.__update_tax_receipt(tax_receipt)
+
+    def __check_BR_requirements(self, tax_receipt, payment_option):
+        # CHECK IF BR TAX AUTHORITY IS CPNJ OR CPF, BECAUSE THEY USE DIFFERENT REQUIREMENTS.
+        if self.__get_epp_tax_identifier_type(payment_option.epp_tax_identifier) == 'CNPJ':
+            for requirement in self.br_requirements + ('recipient_city',):
+                po_attribute = getattr(payment_option, str(self.tax_to_po_requirement_dict.get(requirement)), '')
+                if self.__check_single_requirement(po_attribute):
+                    setattr(tax_receipt, requirement, po_attribute)
+                else:
+                    return
+            self.__update_tax_receipt(tax_receipt)
+
+        elif self.__get_epp_tax_identifier_type(payment_option.epp_tax_identifier) == 'CPF':
+            for requirement in self.br_requirements:
+                po_attribute = getattr(payment_option, str(self.tax_to_po_requirement_dict.get(requirement)), '')
+                if self.__check_single_requirement(po_attribute):
+                    setattr(tax_receipt, requirement, po_attribute)
+                else:
+                    return
+            self.__update_tax_receipt(tax_receipt)
+
+    def __check_single_requirement(self, po_attr):
+        return po_attr == ''
+
+    def __update_tax_receipt(self, tax_receipt):
+        tax_receipt.status_id = TaxReceiptStatuses.get_id_from_name("PENDING")
+        tax_receipt.save(using='billing_local', force_update=True)
+
+
+    def __get_epp_tax_identifier_type(self, epp_tax_identifier): #
+        if len(epp_tax_identifier) > self.CPF_CHAR_COUNT_LIMIT:
+            return 'CNPJ'
+        else:
+            return 'CPF'
+        return ''
+
+
+class TaxReceiptStatuses:
+    @staticmethod
+    def get_id_from_name(string):
+        if string is "INCOMPLETE":
+            return 1
+        elif string is 'PENDING':
+            return 2
```

Comment:
Be careful here, in python `is` compares the memory address that might not be the same. I think that using `==` would be safer since it actually compares the strings (It would be slower tho).